packages/web/components/complex/paginated-table.tsx

```diff
@@ -79,7 +79,10 @@ export const PaginatedTable = ({
           return (
             <Link
               key={row.original.queryPool.id}
+              prefetch={false}
```

Comment:
why is prefetch false? Wouldn't it be better to allow next to prefetch in the background? Or was the pre-fetch causing perf issues?

Reply:
No performance regression in particular, I entered it while I was fixing a re-rendering bug on the pools page, we can reset it to true. Even considering when pools appear with pagination, that's an acceptable amount of prefetches

Reply:
I can change this, if you agree

Reply:
I think it makes sense to do `prefetch={true}`

Reply:
I'll add it

Reply:
I found out, that "prefetch" is also disabled on "stage" branch: https://github.com/osmosis-labs/osmosis-frontend/blob/stage/packages/web/components/cards/pool-card.tsx#L35